gnovm/pkg/gnofmt/processor.go

```diff
@@ -19,7 +19,7 @@
 type parsedPackage struct {
 	error error
 	files map[string]*ast.File
-	decls map[*ast.Object]ast.Decl
+	decls map[*ast.Ident]ast.Decl
```

Comment:
Just a nitpick: what do you think about type aliasing this map?
I see we copypasta it throughout the processor

Reply:
Yes, that would be good. so I aliased as `declMap`